tests/Unit/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Test_TovStar.cpp

```diff
@@ -39,31 +40,45 @@ void verify_solution(const TovStar& solution, const std::array<double, 3>& x) {
   Mesh<3> mesh{brick.initial_extents()[0], Spectral::Basis::Legendre,
                Spectral::Quadrature::GaussLobatto};
   const auto domain = brick.create_domain();
-  verify_grmhd_solution(solution, domain.blocks()[0], mesh, 1.e-10, 1.234,
+  verify_grmhd_solution(solution, domain.blocks()[0], mesh, 1.e-7, 1.234,
                         1.e-4);
 }
 
 void test_tov_star(const TovCoordinates coord_system) {
+  Parallel::printf("Entered the function which tests");
```

Comment:
Probably remove, if not add necessary #include.  Same with the other calls to `Parallel::printf`.